src/plugins/charts/common/palette.ts

```diff
@@ -169,8 +171,14 @@ export function palette(): ExpressionFunctionDefinition<
           range: range ?? 'percent',
           gradient,
           continuity,
-          rangeMin: calculateRange(rangeMin, stops[0], rangeMinDefault),
-          rangeMax: calculateRange(rangeMax, last(stops), rangeMaxDefault),
+          rangeMin:
+            continuity && ['below', 'all'].includes(continuity)
```

Comment:
Can we use the `checkIsMinContinuity` function here?
Adding the existence check to the function can make it usable also here.